src/spikeinterface/sorters/external/mountainsort5.py

```diff
@@ -68,8 +67,8 @@ class Mountainsort5Sorter(BaseSorter):
         "freq_max": "Low-pass filter cutoff frequency",
         "filter": "Enable or disable filter",
         "whiten": "Enable or disable whitening",
-        "temporary_base_dir": "Temporary directory base directory for storing cached recording",
         "n_jobs_for_preprocessing": "Number of parallel jobs for creating the cached recording",
```

Comment:
What's the point of this? If we are removing the create cache couldn't we delete this param and just use the global `n_jobs`?

Reply:
This isn't lining up great I mean delete the `n_jobs_for_preprocessing`?